### DISCUSSION TITLE
Copilot stops working in VS22

### DISCUSSION THREAD

**Discussion by JKamsker** [community/community#45520] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Trigger inline suggestion and open copilot is grayed out
![grafik](https://user-images.githubusercontent.com/11245306/215067318-2ea2e9ad-5108-4a05-bdfe-dababc850f14.png)

GitHub Copilot 1.55.0.0 (v1.55.0.0@a9e44cd78)
GitHub Copilot Agent v1.55.0
[Error] An error occurred for 'RunOperation:GetCompletionsAsync': StreamJsonRpc.RemoteInvocationException: Request getCompletions failed with message: Cannot read properties of undefined (reading 'length')
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__143`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__14`1.MoveNext()

**Comment by jge162** [comment]
You can submit a bug report for more assistance. [bug report](https://support.github.com/contact/bug-report) they will be able to assist you and just be patient..
